lib/commands/performance.js

```diff
@@ -0,0 +1,297 @@
+import { errors } from 'appium-base-driver';
+import { sleep } from 'asyncbox';
+import _ from 'lodash';
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.postDismissAlert = async function () {
+  throw new errors.NotYetImplementedError();
+};
+
+const SUPPORTED_PERFORMANCE_DATA_TYPES = {
+  'cpuinfo': 'the amount of cpu by user and kernel process - cpu information for applications on real devices and simulators',
+  'memoryinfo': 'the amount of memory used by the process - memory information for applications on real devices and simulators',
+  'batteryinfo': 'the remaining battery power - battery power information for applications on real devices and simulators',
+  'networkinfo': 'the network statistics - network rx/tx information for applications on real devices and simulators'
+};
+
+commands.getPerformanceDataTypes = function () {
+  return _.keys(SUPPORTED_PERFORMANCE_DATA_TYPES);
+};
+
+commands.getPerformanceData = async function (applicationPackageName, performanceDataType, performanceDataReadTimeout) {
```

Comment:
In the context of the function, there is no need for "application" in the parameter names. It would be more succinct and readable to have

```js
commands.getPerformanceData = async function (packageName, dataType, dataReadTimeout) {
  // ...
```